industrial_extrinsic_cal/include/industrial_extrinsic_cal/ros_camera_observer.h

```diff
@@ -233,11 +233,11 @@ namespace industrial_extrinsic_cal
     /**
      *  @brief getImageNumber 
      */
-    int getImageNumber(){ return(image_number_);};
+    int getImageNumber(){ return(image_number_);}
```

Comment:
It is unnecessary since the function is defined at the same time as it is declared.